Venter/models.py

```diff
@@ -5,6 +5,14 @@
 
 
 class Organisation(models.Model):
+    """
+    An organisation that the user belongs to.
+    Eg: user_1 belongs to xyz organisation
+
+    # Create an organisation
+    >>> organisation_1 = Organisation.objects.create(organisation_name="xyz", organisation_logo="image1.png")
+    >>> organisation_2 = Organisation.objects.create(organisation_name="abc", additional_details="Mumbai based company")
```

Comment:
You don't need to write sample code here. Also, you could use this docstring to explain the components of the model class instead of explaining the underlying code,

Reply:
@simranmadhok  Let these comments be for the time being, 
@samjakes we can consider removing the sample code during the wrap-up/final review of the code
I am suggesting thus because the sample code would be helpful to all the different people accessing the code base.

Reply:
@samjakes Requesting you to approve these changes as is.